android/tests/espresso/com/frostwire/android/test/utils/PreferencesManipulator.java

```diff
@@ -0,0 +1,42 @@
+package com.frostwire.android.test.utils;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+
+import com.frostwire.android.core.Constants;
+
+import java.util.Map;
+
+public class PreferencesManipulator {
+    SharedPreferences mPreferences;
```

Comment:
make this:
`private final SharedPreferences mPreferences;`

This way you will know for a fact that the `mPreferences` instance will never be `null` and that nobody will be able to change it.